wd_helper.au3

```diff
@@ -2891,7 +2891,7 @@ EndFunc   ;==>_WD_DispatchEvent
 ;                  - $_WD_ERROR_NoMatch
 ; Author ........: danylarson
 ; Modified ......: water, danp2, mLipok
-; Remarks .......:
+; Remarks .......: This function also returns nested table rows. If you want to prevent this, modified $sRowsSelector can be used e.g. ":scope > tbody > tr"
```

Comment:
I prefer something like this --

The CSS selectors can be overridden to control the included elements. For example, a modified $sRowsSelector of ":scope > tbody > tr" can be used to bypass nested tables.

Reply:
Done